openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +181,85 @@ definitions:
       created:
         type: string
         format: date-time
-        description: |-
+        description: >-
           Timestamp of object creation in RFC3339.
       updated:
         type: string
         format: date-time
-        description: |-
-          Timestamp of update in RFC3339, identical to create timestamp in systems
-          that do not support updates.
+        description: >-
+          Timestamp of update in RFC3339, identical to create timestamp in
+          systems that do not support updates.
       version:
         type: string
-        description: |-
-          A string representing a version, some systems may use checksum, a RFC3339
-          timestamp, or incrementing version number. For systems that do not support
-          versioning please use your update timestamp as your version.
+        description: >-
+          A string representing a version, some systems may use checksum, a
+          RFC3339 timestamp, or incrementing version number. For systems that do not
+          support versioning please use your update timestamp as your version.
+      mime-type:
+        type: string
+        description: >-
+          A string providing the mime-type of the Data Bundle.
+          For example, "application/json".
       checksums:
         type: array
+        description: >-
+          At least one checksum must be provided.
+          The Data Bundle checksum is computed over a sorted concatenation of all the 
+          checksums of the Data Objects that bundle contains. The list of Data
+          Object checksums are sorted alphabetically (hex-code) before concatenation and a further
+          checksum performed on the concated binary value. Example below:
+          Data Ojects:
+            md5(DO1) = 72794b6d30bc86d92e40a1aa65c880b8
+            md5(DO2) = 5e089d29a18954e68a78ee6a3c6edabd
+          Data Bundle:
+          DB1 = md5( concat( sort( md5(DO1), md5(DO2) ) ) )
+              = md5( concat( sort( 72794b6d30bc86d92e40a1aa65c880b8, 5e089d29a18954e68a78ee6a3c6edabd ) ) )
+              = md5( concat( 5e089d29a18954e68a78ee6a3c6edabd, 72794b6d30bc86d92e40a1aa65c880b8 ) )
+              = md5( 5e089d29a18954e68a78ee6a3c6edabd72794b6d30bc86d92e40a1aa65c880b8 )
+              = f7a29a0422e7d870b10839ad6c985079
         items:
           $ref: '#/definitions/Checksum'
-        description: |-
-          At least one checksum must be provided.
-          The Data Bundle checksum is computed over all the checksums of the
-          Data Objects that bundle contains.
+      access_methods:
+        type: array
+        items:
+          $ref: '#/definitions/AccessMethod'
       description:
         type: string
-        description: |-
-          A human readable description.
+        description: A human readable description.
       aliases:
         type: array
+        description: A list of strings that can be used to identify this Data Bundle.
```

Comment:
Probably need to incorporate the changes to align with our discussion in the meeting today.

Reply:
And to align with our further discussion in the meeting **today**.

Reply:
Or, since this PR doesn't actually change the text of the description, you could just remove the syntax change to the YAML, and then this edit won't conflict with the one in #246 -- you call which is easier.

Reply:
I changed the `aliases` description to align with #246